src/core/server/dev/dev_config.ts

```diff
@@ -34,7 +34,7 @@ export const config = {
   path: 'dev',
   schema: schema.object({
     basePathProxyTarget: schema.number({
-      defaultValue: 5603,
+      defaultValue: 6603,
```

Comment:
Is this port updated by accident?

Reply:
Thank you for your catch. It's my mistake.